README.md

```diff
@@ -45,6 +37,18 @@ for user in json.loads(client.user.list())['users']:
     print client.meeting.list('host_id': user_id)
 ```
 
+### Create the client v2
+
+```python
+from zoomus import ZoomClient
+
+client = ZoomClient('API_KEY', 'API_SECRET', version=2)
```

Comment:
Should we make V2 the default? Yes this may break some scripts that people have, but V1 has been deprecated and most people should have switched over?

Reply:
I think a breaking change like that should require a major version bump. But, given that this is currently a pre-1.0 product, I think that would be acceptable, saying that the publicly defined API is V2 by default.

But I also think that in the future, removing Py2 support should require a major version bump. So it comes down to if you want that in 1.0.0 or 2.0.0.

Reply:
Let's make V2 the default in release 1.0 and then deprecate Py2 in 2.0. We'll just make sure to call it out in the release notes.

Reply:
@prschmid So does that mean you would like to move this PR to V2 as the default, and bump the version to 1.0? Or keep this PR as-is, with a follow-up 1.0 release with the switch to V2?

If the former, just let me know what else I need to do to get this thing merged in. 😄

Reply:
Sorry for the confusion. Let's keep the PR as is, so that once this is merged in to develop the V2 Zoom API will be the default. I will then create a new release of the client and label it as the V1 release of the zoomus library.

Reply:
Great! Then I'm ready whenever you are! 😄